src/diagnostic-channel-publishers/src/mysql.pub.ts

```diff
@@ -101,7 +101,7 @@ const mysqlPatchFunction: PatchFunction = function(originalMysql, originalMysqlP
 };
 
 export const mysql: IModulePatcher = {
-    versionSpecifier: ">= 2.0.0 <= 2.14.0",
+    versionSpecifier: ">= 2.0.0 <= 2.14.x",

```

Comment:
there's a big test gap here in that we should probably be testing against all supported minor revs we support in the version specifier.  i.e., we should be validating against mysql 2.0, 2.1, 2.2,...